sofl/game_data.py

```
@@ -0,0 +1,136 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import shlex
+from pathlib import Path
+from time import time
+from typing import Any, Optional, Callable
```

Flake8 (7.2.0)

[error] 'typing.Callable' imported but unused

(F401)